pytorch_toolkit/ote/ote/tests/test_case.py

```diff
@@ -139,22 +152,25 @@ def do_finetuning(self, on_gpu):
 
             self.assertTrue(os.path.exists(os.path.join(self.output_folder, 'latest.pth')))
 
-        @unittest.skipUnless(torch.cuda.is_available(), 'No GPU found')
         def test_evaluation_on_gpu(self):
+            skip_if_cuda_not_available()
             self.do_evaluation(on_gpu=True)
 
         def test_evaluation_on_cpu(self):
             skip_if_cpu_is_not_supported(self.template_file)
             self.do_evaluation(on_gpu=False)
 
-        @unittest.skipUnless(torch.cuda.is_available(), 'No GPU found')
         def test_finetuning_on_gpu(self):
+            skip_if_cuda_not_available()
             self.do_finetuning(on_gpu=True)
 
         def test_finetuning_on_cpu(self):
             skip_if_cpu_is_not_supported(self.template_file)
             self.do_finetuning(on_gpu=False)
 
+        def test_smoke_test(self):
```

Comment:
what is it for?

Reply:
It is really useful for quick checks of ote if it is buildable -- now we can set the command line parameter `--test-id-filter "*smoke*" ` for `tests/run_model_templates_tests2.py` to run these tests only